examples/src/CompactServer.tsx

```diff
@@ -13,8 +13,8 @@ export class Index extends React.Component<any, any> {
         return (
             <div>
                 <CompactServer id={'CUSTOM-PC.localdomain'} onClose={this._onServerCloseCompactServer} onRoleEdit={this._onClickCompactServer} name={'CUSTOM-PC'} roles={[]} status={1} />
-                <CompactServer id={'My very very long name of a server I am using I know its very long.domain.com' } onClose={this._onServerCloseCompactServer} onRoleEdit={this._onClickCompactServer} name={'My very very long name of a server I am using I know its very long'} roles={[]} status={2} />
-                <CompactServer id={'BANANA-PC.banana.com' } onClose={this._onServerCloseCompactServer} onRoleEdit={this._onClickCompactServer} name={'BANANA-PC'} roles={[{ display: 'WPF', iconName: 'icon-add' }, { display: 'Search', iconName: 'icon-alert' }]} status={0} />
+                <CompactServer id={'My very very long name of a server I am using I know its very long.domain.com' } onRoleEdit={this._onClickCompactServer} name={'My very very long name of a server I am using I know its very long'} roles={[]} status={2} />
+                <CompactServer id={'BANANA-PC.banana.com' } onRoleEdit={this._onClickCompactServer} onClose={this._onServerCloseCompactServer} name={'BANANA-PC'} roles={[{ display: 'WPF', iconName: 'icon-add' }, { display: 'Search', iconName: 'icon-alert' }]} status={0} />
```

Comment:
reading this  *onRoleEdit={this._onClickCompactServer}* confuses me a bit. Are we editing roles, or are we clicking the server?